spec/example_app/config/database.yml

```diff
@@ -1,4 +1,5 @@
 development: &default
+  host: localhost
```

Comment:
Was this necessary to get up and running? Should we include this in this PR, or elsewhere?